examples/example_HSC_DRP-RC2.yaml

```diff
@@ -19,7 +19,7 @@
                 spec_block: HSC_DRP-RC2_step1
                 child_config:
                     split_method: split_by_query
-                    split_min_groups: 5
+                    split_min_groups: 3
```

Comment:
PR title says "reduce from 5 to 2" -- did you intend 3 here and below?  If so, go ahead and edit the PR title to 3 please!